### ISSUE TITLE
TextInput's text overlap with label when number of lines > 5 (ios)

### ISSUE THREAD

**Issue by HanShenNg** [callstack/react-native-paper#1306]
### Current behaviour

TextInput's text overlaps with its label when number of lines > 5.

### Expected behaviour

The text in the TextInput shouldn't overlap with its label.

### Screenshots

![image](https://user-images.githubusercontent.com/32507141/63745776-a4347000-c8d5-11e9-8a31-c67f98fdd1e4.png)
Original layout (focused on affected field)

![image](https://user-images.githubusercontent.com/32507141/63745894-ed84bf80-c8d5-11e9-855c-0cf8d1f2d77a.png)
TextInput with 5 lines

![image](https://user-images.githubusercontent.com/32507141/63745909-f83f5480-c8d5-11e9-9921-a11dd5e54217.png)
TextInput with 6 lines (text overlap with label)

![image](https://user-images.githubusercontent.com/32507141/63746011-3d638680-c8d6-11e9-9ca4-32b33d68ccfe.png)
Android: TextInput with 6 lines

### What I have tried

1. Tweak the values of numberOfLines, zIndex and elevation props.
2. Uninstall and reinstall react native paper package.
3. Test with both ios and android emulators.

### Your Environment

| software              | version
| --------------------- | -------
| ios        | 12.4
| react-native          | 0.60.3
| react-native-paper    | 2.16.0
| node                  | 10.16.0
| npm           | 6.10.0

**Comment by jbinda** [comment]
Hi @HanShenNg 

The issue is connected with #974 and it will be fixed in 3.0 version after deal with some other tasks. Please consider to migrate to 3.0 because I see you are on 2.16 version

**Comment by LucasClaude** [comment]
I used this as a temporary solution until it's properly fixed (NativeTextInput is just the standard react-native text input that React Paper renders by default):

```
 render={(props: RenderProps) => (
     <NativeTextInput
         {...props}
         style={[props.style, { paddingTop: 5, marginTop: 5 }]}
     />
 )}
```
